Add-on Documentation/Data Processing/QuotaGuardStatic.md

```diff
@@ -49,7 +49,7 @@ res = RestClient.get("http://ip.jsontest.com")
 puts "Your Static IP is: #{res.body}"
 ~~~
 
-Alternatively you can read the QuotaGuard Static credentials from the creds.json file. The location of the file is available in the `CRED_FILE` environment variable. For detailed instructions on how to use the creds.json file please refer to the section about [Add-on Credentials](https://www.cloudcontrol.com/dev-center/Platform%20Documentation#Add-Ons) in the general documentation.
+Alternatively you can read the QuotaGuard Static credentials from the creds.json file. The location of the file is available in the `CRED_FILE` environment variable. For detailed instructions on how to use the creds.json file please refer to the section about [Add-on Credentials](https://www.cloudcontrol.com/dev-center/platform-documentation#Add-Ons) in the general documentation.
```

Comment:
The `#Add-Ons` needs to be lower-case here too. Like this: `platform-documentation#add-ons`